app/controllers/invites_controller.rb

```diff
@@ -2,12 +2,10 @@ class InvitesController < ApplicationController
 
   def create
     @invite = Invite.new(invite_params)
-    if @invite.save
-      render json: { invite: @invite, message: "Success" }
-    else
-      #TODO: For temp message display
-      render json: { message: @invite.errors[:user_id][0] }
-    end
+    @event = Event.find(params[:invite][:event_id])
+    @room = @event.room
+    @campus = @event.campus
+    save_for_html_json(@invite, "/events/show") { :back }
```

Comment:
This refactoring breaks the "Success", and various failure messages that show up after an invite is sent. I suggest just adding a respond to html if thats what you need for users joining open events functionality.